ArrayOperations/Demo.cs

```diff
@@ -0,0 +1,111 @@
+using System;
+
+namespace ArrayOperations
+{
+  public static class Demo
+  {
+    public static void Run()
+    {
+      var input = new[] {1, 3, 5, 6, 7, 10, 8, 2};
+      
+      try
+      {
+        SortArray(input);
+        Pause();
+        
+        InsertAnElementAtBeginningOfTheArray(input);
+        Pause();
+
+        InsertAnElementAtEndOfArray(input);
+        Pause();
+
+        RemoveFirstElementOfArray(input);
+        Pause();
+
+        RemoveLastElementOfArray(input);
+        Pause();
+
+        InsertElementAtGivenIndex(input);
+        Pause();
+
+        RemoveElementAtGivenIndex(input);
+        Pause("Press ENTER to exit...");
+      }
+      catch (Exception ex) when (ex is ArgumentException 
+                                 || ex is ArgumentNullException 
+                                 || ex is ArgumentOutOfRangeException)
+      {
+        Console.WriteLine(ex.Message);
+      }
+    }
+
+    private static void SortArray(int[] input)
+    {
+      Arrays.Print(input, "Initial array elements:");
+      var output = Arrays.Sort(input);
+      Arrays.Print(output, "Array after sort operation:");
+    }
+
+    private static void InsertAnElementAtBeginningOfTheArray(int[] input)
+    {
+      Arrays.Print(input,"Initial array elements:");
+
+      const int numberToInsert = 12;
```

Comment:
Use `NumberToInsert` (first letter capitalized) for constants.

Reply:
By default Visual Studio and Rider recommend to use camel case and it stuck with me.